specs/README.md

```diff
@@ -0,0 +1,9 @@
+# Specs
+
+Here you'll find specifications for fast-dna custom elements and other library features.
+
+> **Note:** When authoring a new spec, please use [our standard spec template](./template.md). The PR that introduces your spec should also add a link to this readme in the specs section below.
+
+## Specs
+
+Coming soon...
```

Comment:
for web content, generally, I'm not a fan of "coming soon".  I would rather than add it when it's ready.